chef/cookbooks/repos/recipes/default.rb

```diff
@@ -30,34 +37,35 @@
 
   case node[:platform_family]
   when "debian"
-    repositories = provisioner["provisioner"]["repositories"][os_token][arch]
-    cookbook_file "/etc/apt/apt.conf.d/99-crowbar-no-auth" do
-      source "apt.conf"
-    end
-    file "/etc/apt/sources.list" do
-      action :delete
-    end
-    repositories.each do |repo,urls|
-      case repo
-      when "base"
-        template "/etc/apt/sources.list.d/00-base.list" do
-          variables(urls: urls)
-          notifies :create, "file[/tmp/.repo_update]", :immediately
-        end
-      else
-        template "/etc/apt/sources.list.d/10-barclamp-#{repo}.list" do
-          source "10-crowbar-extra.list.erb"
-          variables(urls: urls)
-          notifies :create, "file[/tmp/.repo_update]", :immediately
-        end
-      end
-    end
-    bash "update software sources" do
-      code "apt-get update"
-      notifies :delete, "file[/tmp/.repo_update]", :immediately
-      only_if { ::File.exists? "/tmp/.repo_update" }
-    end
-    package "rubygems"
+# FIXME: need to rewrite this
+#    repositories = provisioner["provisioner"]["repositories"][os_token][arch]
+#    cookbook_file "/etc/apt/apt.conf.d/99-crowbar-no-auth" do
+#      source "apt.conf"
+#    end
+#    file "/etc/apt/sources.list" do
+#      action :delete
+#    end
+#    repositories.each do |repo,urls|
+#      case repo
+#      when "base"
+#        template "/etc/apt/sources.list.d/00-base.list" do
+#          variables(urls: urls)
+#          notifies :create, "file[/tmp/.repo_update]", :immediately
+#        end
+#      else
+#        template "/etc/apt/sources.list.d/10-barclamp-#{repo}.list" do
+#          source "10-crowbar-extra.list.erb"
+#          variables(urls: urls)
+#          notifies :create, "file[/tmp/.repo_update]", :immediately
+#        end
+#      end
+#    end
+#    bash "update software sources" do
+#      code "apt-get update"
+#      notifies :delete, "file[/tmp/.repo_update]", :immediately
+#      only_if { ::File.exists? "/tmp/.repo_update" }
+#    end
+#    package "rubygems"
```

Comment:
Style/CommentIndentation: Incorrect indentation detected (column 0 instead of 4).